apps/ui-kit/src/lib/components/organisms/dialog/Dialog.tsx

```diff
@@ -41,19 +41,24 @@ const DialogOverlay = React.forwardRef<
 >(({ showCloseIcon, position, ...props }, ref) => (
     <RadixDialog.Overlay
         ref={ref}
-        className={cx('dialog-overlay-bg inset-0 z-[99998] backdrop-blur-md', {
-            fixed: position === DialogPosition.Right,
-            absolute: position === DialogPosition.Center,
-        })}
+        className={cx(
+            ' dialog-overlay-bg absolute h-full w-full backdrop-blur-md names:backdrop-blur-lg',
+        )}
         {...props}
     >
         <DialogClose className={cx('fixed right-3 top-3', { hidden: !showCloseIcon })}>
-            <Close />
+            <Close className="button-text-color-neutral" />
         </DialogClose>
     </RadixDialog.Overlay>
 ));
 DialogOverlay.displayName = RadixDialog.Overlay.displayName;
 
+const DialogContainer = React.forwardRef<HTMLDivElement, React.PropsWithChildren>((props, ref) => (
+    <div className="fixed inset-0 z-[99999]" ref={ref}>
+        <div className="relative h-full w-full">{props.children}</div>
+    </div>
+));
+
```

Comment:
This is triggering an error in the names dapp

Reply:
can you check again, this is supposed to be fixed